tests/ts-app-template/types/ts-app-template/index.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="@embroider/core/virtual" />
```

Comment:
this is the only way I could get the types present when type-checking.

The (imo) nicer way, via `compilerOptions.types` doesn't seem to work for @-prefixed package names.